src/assets/_project/_blocks/components/link/_qg-link.scss

```diff
@@ -9,7 +9,7 @@
   // make exception for certain classes, which need to re-visit for better optimization
   &:not(.qg-feature-box .qg-link):not(.qg-index a.qg-index-item):not(.page-item
       .page-link):not(.dfv-cards--type-top-prompt
-      a.card):not(.search-categories__index-toggle):not(.search-categories__index):not(.QSAR-records-search a) {
+      a.card):not(.search-categories__index-toggle):not(.search-categories__index):not(.QSAR-records-search a):not(.acc-heading a) {
```

Comment:
This causes links to miss out on the underline offset.

Reply:
Yes right. we did not need active state style, because the header box gets the out line. But we needed default and hover state. Updated now